src/api.rs

```diff
@@ -105,6 +104,14 @@ generate_enums! {
     DebugDumpStore: 0x79
 }
 
+pub trait RequestData: Into<Request> + TryFrom<Request, Error = crate::Error> {
```

Comment:
`RequestVariant` and `ReplyVariant`? Would more precisely name what we expect.

Reply:
done in ef3deac6744a6ec231a7cbd366fbbce19f4d84d0